### ISSUE TITLE
Pre-trained weights

### ISSUE THREAD

**Issue by SupreethN** [gkakogeorgiou/spot#1]
Thank you so much for open-sourcing this fantastic work! Is there any plan to release the pre-trained weights and a script to visualize the slot-attention masks?

**Comment by gkakogeorgiou** [comment]
Thank you for your interest and kind words about our work! 

We have now added the pre-trained models for COCO and VOC datasets (https://github.com/gkakogeorgiou/spot/tree/master?tab=readme-ov-file#pretrained-model and https://github.com/gkakogeorgiou/spot/tree/master?tab=readme-ov-file#pretrained-model-1 ). 

The script for visualizing the slot-attention masks still needs to be polished for release, as it is also part of another project.
Nevertheless, we have a section within our training script that might be useful for visualization: https://github.com/gkakogeorgiou/spot/blob/master/train_spot.py#L339-L346
We hope this helps you.
